optuna/terminator/erroreval.py

```diff
@@ -27,11 +27,33 @@ def evaluate(
 
 @experimental_class("3.2.0")
 class CrossValidationErrorEvaluator(BaseErrorEvaluator):
+    """An error evaluator for objective functions based on cross-validation.
+
+    This evaluator evaluates objective functions' statistical error, which comes from the
```

Comment:
```suggestion
    This evaluator evaluates the objective function's statistical error, which comes from the
```
Only one objective function is used in most cases.